### PR TITLE
Refactoring GCP credentials -Fix continuation

### PR DESCRIPTION
**Pull request by animehart**
On my previous PR (https://github.com/elastic/integrations/pull/7537) I forgot to include the changelog and manifest yml change
as such It didnt trigger the build to see my changes

### CODE REVIEW

File: packages/cloud_security_posture/changelog.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@
 # 1.4.x - 8.9.x
 # 1.3.x - 8.8.x
 # 1.2.x - 8.7.x
+- version: "1.5.2-preview2"
+  changes:
+    - description: Refactor GCP credentials -FIX
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/7537
```
Comment (mitodrummer) [inline packages/cloud_security_posture/changelog.yml:11]: update PR to 7546

File: packages/cloud_security_posture/changelog.yml [hunk 2]
```diff
@@ -4,6 +4,11 @@
 # 1.4.x - 8.9.x
 # 1.3.x - 8.8.x
 # 1.2.x - 8.7.x
+- version: "1.5.2-preview2"
+  changes:
+    - description: Refactor GCP credentials -FIX
```
Comment (opauloh) [inline packages/cloud_security_posture/changelog.yml:9]: can you change the description to `Refactor GCP credentials - Bump Version`?

**Review by mitodrummer — APPROVED** [review]
LGTM!

**Review by opauloh — APPROVED** [review]

**Comment by oren-zohar** [comment]
@animehart was it tested with `preview`? Can we publish it in production?

**Pull request merged**